invoicer_project/static/js/utils_clients.js

```diff
@@ -0,0 +1,209 @@
+export const host = "http://127.0.0.1:8000";
```

Comment:
I think, that this file should have other name than `utils_clients.js`. It is good idea to move general functions to one place, but let's make next step to improve our code. Let's gather functions related to validations to one place, functions related to interaction with server - to another place and other functions could be left here.

Reply:
Done